### PR TITLE
Fit Criterion implemented & VDM refactored

### PR DESCRIPTION
**Pull request by notantony**

### CODE REVIEW

File: filters/FitCriterion.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+            # Main calculations
+            for sample_index, value in enumerate(feature):  # For each sample value
+                for i in range(tokensN):  # For each class token
+                    ##TODO: ?? cases 0 / 0, smth / 0
+                    distances[i] = np.abs(value - centers[i]) / variances[i]
+                fc[feature_index] += np.argmin(distances) == y[sample_index]
+
+        fc /= y.shape[0]  # Normalization
+        print(fc)
```
Comment (LastShekel) [inline filters/FitCriterion.py]:
```suggestion
        return fc
```
We should return dictionary of values {feature_name : measure_value}
So you should attach feature names, take a look at generate_features function in data_check.py file

File: filters/FitCriterion.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        tokensN = np.max(y) + 1  # Number of different class tokens
+
+        # Utility arrays
+        centers = np.empty(tokensN)  # Array with centers of sets of feature values for each class token
+        variances = np.empty(tokensN)  # Array with variances of sets of feature values for each class token
+        # Each of arrays above will be separately calculated for each feature
+        distances = np.empty(tokensN) # Array with distances between sample's value and each class's center
+        # This array will be separately calculated for each feature and each sample
+
+        for feature_index, feature in enumerate(x.T):  # For each feature
```
Comment (LastShekel) [inline filters/FitCriterion.py]: doesn't work if i use pandas dataframe as X 
try to replace enumerate with something else

File: filters/VDM.py [hunk 3]
```diff
@@ -1,94 +1,100 @@
...
+            Returns
+            -------
+            result:
+                numpy.ndarray, shape (n_samples, n_samples) with selected version of metrics
+            See Also
+            --------
+        """
         vdm = np.zeros((x.shape[0], x.shape[0]))  # Initializing output matrix
 
         for column in tqdm(x.T):  # For each attribute separately:
```
Comment (LastShekel) [inline filters/VDM.py]: Thats doesn't work with pandas dataframe too(

Comment (notantony) [reply filters/VDM.py]: fixed

File: filters/FitCriterion.py (hunk 2 above)
Comment (notantony) [reply filters/FitCriterion.py]: fixed

**Pull request merged**